src/qt/transactionfilterproxy.cpp

```diff
@@ -8,11 +8,12 @@
 #include <qt/transactionrecord.h>
 
 #include <cstdlib>
+#include <limits>
 
 // Earliest date that can be represented (far in the past)
-const QDateTime TransactionFilterProxy::MIN_DATE = QDateTime::fromTime_t(0);
+const QDateTime TransactionFilterProxy::MIN_DATE = QDateTime::fromSecsSinceEpoch(0);
 // Last date that can be represented (far in the future)
-const QDateTime TransactionFilterProxy::MAX_DATE = QDateTime::fromTime_t(0xFFFFFFFF);
+const QDateTime TransactionFilterProxy::MAX_DATE = QDateTime::fromSecsSinceEpoch(std::numeric_limits<qint64>::max());
```

Comment:
From Qt documentation [0]:

> Note that there are possible values for secs that lie outside the valid range of QDateTime, both negative and positive. The behavior of this function is undefined for those values.

Anyone knows what are these "ouside" values, and is this `::max()` OK here then?

[0] https://doc.qt.io/qt-5/qdatetime.html#fromSecsSinceEpoch

Reply:
I guess we have to wait for https://github.com/bitcoin-core/gui/pull/354 to be merged, these lines will go away.